app/controllers/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+require 'pry'
+
+class UsersController < ApplicationController
+
+  get '/users/:slug' do
+    @user = User.find_by_slug(params[:slug])
+    erb :'users/show'
+
+  end
+
+  get '/signup' do
+    if !session[:user_id]
```

Comment:
Hey Lukel! thanks for these! I will take care of these within next 24hrs and will let you know when they are changed.

~bdc
On Aug 31, 2016, at 11:33 AM, Luke Ghenco <notifications@github.com<mailto:notifications@github.com>> wrote:


In app/controllers/users_controller.rb<https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/197#discussion_r77010783>:

> @@ -0,0 +1,59 @@
> +require 'pry'
> +
> +class UsersController < ApplicationController
> +
> +  get '/users/:slug' do
> +    @user = User.find_by_slug(params[:slug])
> +    erb :'users/show'
> +
> +  end
> +
> +  get '/signup' do
> +    if !session[:user_id]


if !logged_in?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/learn-co-students/sinatra-fwitter-group-project-v-000/pull/197/files/22c0e06ca974b756168a09b6b5003de7689f012a#r77010783>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AO33F-q2mM-SxI_90eMyj7YPUYH_e4G7ks5qlZ7GgaJpZM4JwfyH>.